client/src/components/Form.js

```diff
@@ -0,0 +1,123 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
```

Comment:
```suggestion
import styled from 'styled-components/macro';
```

See: https://styled-components.com/docs/tooling#babel-macro